src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -45,6 +44,21 @@ pub struct IngestAction {
     omit_value: bool,
 }
 
+impl IngestAction {
+    /// Whether the action causes a schema to be published
+    /// to CSR
+    pub fn publish(&self) -> bool {
+        match &self.format {
+            Format::Avro {
+                confluent_wire_format,
+                ..
+            } => *confluent_wire_format,
+            Format::Protobuf { .. } => false,
```

Comment:
proto can use csr too

Reply:
We never publish Proto schemas to the CSR in testdrive AFAICT.

I will let someone who understands the semantics of Protobuf and how it interacts with the CSR try to figure out why that's true; this PR is scoped only to Avro.

Reply:
Protobuf schemas are split across multiple files. There's no easy way to have the equivalent of the `schema` option that we have for Avro-formatted records.